L/LLVM/common.jl

```diff
@@ -249,6 +249,9 @@ if [[ "${target}" == *darwin* ]]; then
     ln -s libLLVM.dylib ${prefix}/lib/libLLVM-${LLVM_VER##*-}.dylib
 fi
 
+# Validate llvm-config works
+${prefix}/tools/llvm-config --version
```

Comment:
We can't do this since we are in BinaryBuilder and we can't execute tools build for the target

Reply:
Ok, it's a target executable, thanks for confirming.  I also seen it's listed among products, so there should be some minimal sanity check already.

Reply:
Ah yes, that makes sense. It's too bad we can't validate in BinaryBuilder though

Reply:
BTW, I discovered that for executables the "minimal sanity checks" are only "the file exists". Fairly minimal